html/webappapis/timers/clearinterval-from-callback.any.js

```diff
@@ -0,0 +1,19 @@
+async_test((t) => {
+  let timesCalled = 0;
```

Comment:
Nit: maybe make this a boolean to make it more clear it's solely intended to be a guard.